.gitignore

```diff
@@ -7,4 +7,7 @@ dist/*
 .venv
 .env/
 Pipfile
-*.egg-info
\ No newline at end of file
+*.egg-info
```

Comment:
Backends broadly speaking have two main features: state locking and remote state storage. Locking prevents two executions happening at the same time. And remote state storage allows you to put your state in a...

📄 [Open Up The Cloud](https://openupthecloud.com/terraform-best-practices/)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I have no idea. 
The software constructs all or part of an OS command using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended OS command when it is sent to a downstream component.

📄 [Mitre](https://cwe.mitre.org/index.html)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Love this code! It's awesome and elegant.
Do not prefix stored procedures with sp_, because this prefix is reserved for identifying system-stored procedures;
In Transact-SQL, do not prefix variables with @@, which should be reserved for truly global variables such as @@IDENTITY.

📄 [Sabia](http://sabia.tic.udc.es/docencia/is/docs/documentos/CSharpCodeStyleGuide.pdf)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Elegant

Reply:
test 3/31/2022